src/window_battlestatus.cpp

```diff
@@ -155,7 +159,9 @@ void Window_BattleStatus::RefreshGauge() {
 				int y = 2 + i * 16;
 
 				if (lcf::Data::battlecommands.battle_type == lcf::rpg::BattleCommands::BattleType_alternative) {
-					DrawGauge(*actor, 202 - 10, y - 2);
+					if (lcf::Data::battlecommands.transparency == lcf::rpg::BattleCommands::Transparency_opaque || (2 + index * 16 != y)) {
```

Comment:
What is the purpose of the ``2 + index * 16 != y`` check?

Reply:
This check stops the gauge (if the gauge is opaque) being drawn if the menu cursor is on the same position.

Reply:
I thought about this now for 15 minutes but now I fully understand why they did this (typical 2k3 half assed job):

The gauge is drawn transparent but above the selection box. This means that the color of the gauge would be wrong because of the selection box as the draw order is "Background, Selection Box, Gauge, Text".

Instead of thinking about a solution they simply hid the bug. Genious move -_-.

Could you add a comment there:
```
RPG_RT Bug (?): Gauge hidden when selected due to transparency (wrong color)
```
Or something similiar.

Reply:
Added.